WordPress/build.gradle

```diff
@@ -717,19 +717,22 @@ if (project.hasProperty("debugStoreFile")) {
 // Copy React Native JavaScript bundle and source map so they can be upload it to the Crash logging
 // service during the build process.
 android {
-    applicationVariants.all { variant ->
-        variant.mergeAssetsProvider.configure {
-            doLast {
-                // Copy bundle and source map files
-                copy {
-                    from(outputDir)
-                    into("${buildDir}/react-native-bundle-source-map")
-                    include("*.bundle", "*.bundle.map")
-                }
+    applicationVariants.configureEach { variant ->
+        def variantAssets = "${buildDir}/intermediates/assets/${variant.name}"
```

Comment:
I think we can also get this path with the following code: `variant.mergeAssetsProvider.get().outputDir.get()`.

Reply:
You're right, it's better to not hardcode it. Fixed in 6ea143cef4e471a198fb8a192367e22e69e8962f